### ISSUE TITLE
Warn the user about missing files

### ISSUE THREAD

**Issue by teplit** [Beep6581/RawTherapee#5793]
If the PP3 file points to a missing DCP profile, RawTherapee 5.8 will not notify the user about that and the latter may not notice the problem indicated simply as a **(None)** Input Profile. This is potentially unsafe situation.

**Comment by teplit** [comment]
More generally RawTherapee can display a warning about missing user specified file(s) during reading of PP3 file. The user specified files (e.g. DCP profile, LCP file and others) affect the image processing and their silent substitution can change the intended look of the image and cause subsequent problems if the user don’t notice their absence. Some widely used industrial applications not only warn the user about missing files, but offer him/her a window to locate them before starting work.

**Comment by Beep6581** [comment]
Duplicate of #4261
